packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -872,15 +872,17 @@ Content-Type: application/json
 Include the `X-Device-Fingerprint` header to supply a device fingerprint. The `X-Device-Fingerprint` header is used in the following ways:
 
 * If the new or unknown device email notification is enabled, an email is sent to the user if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?).
-* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see [EA documentation](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm?).
+* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent in the `X-Device-Fingerprint` header isn't associated with a previously successful user sign in. For more information about this feature, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
+
+> **Note:** Okta is improving its new device behavior detection such that it will no longer use a device fingerprint. This change will be behind a self-service feature flag TODO ADD FRIENDLY NAME HERE available June 15th, 2020. Customers relying on device fingerprinting for new device behavior detection should start sending a `deviceToken` in the [context object](#context-object) to uniquely identify devices for behavior detection. Starting September 1, 2020 the update will be made for all orgs and the device fingerprint will no longer be accepted as a valid device identifier for new device behavior detection. Other features relying on device fingerprinting will not be affected at this time. For more information, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
```

Comment:
self service FFs have "friendly names". The feature is not yet marked self service with a friendly name so I left a place holder